documentation/docs/getting_started/getting_started.md

```diff
@@ -64,42 +68,42 @@ Hornet ships as a single executable binary (`hornet` or `hornet.exe`) and some J
 
 ### Linux (and Raspberry Pi)
 
-Recommended installation methods:
+Hornet on Linux can be installed using:
 
-- [Hornet apt repository](hornet_apt_repository.md#hornet-apt-repository-linux-distro-specific)
-- [Docker image](using_docker.md)
+- [The hornet apt repository](https://wiki.iota.org/hornet/getting_started/hornet_apt_repository).
+- [The docker image](https://wiki.iota.org/hornet/getting_started/using_docker).
 
-Other installation methods:
+It can also be installed using:
 
-- [Prebuilt binary files](hornet_apt_repository.md#pre-built-binaries)
-- [Build from source](hornet_apt_repository.md#build-from-source)
+- [Prebuilt binary files](hornet_apt_repository.md#pre-built-binaries), or
+- [Built from the source](hornet_apt_repository.md#build-from-source).
 
 ### MacOS
 
-Recommended installation methods:
+Hornet on MacOS can be installed using:
 
-- [Docker image](using_docker.md)
+- [The docker image](https://wiki.iota.org/hornet/getting_started/using_docker).
 
-Other installation methods:
+It can also be installed using:
 
-- [Prebuilt binary files](hornet_apt_repository.md#pre-built-binaries)
+- [Prebuilt binary files](https://wiki.iota.org/hornet/getting_started/using_docker#starting-an-existing-hornet).
 
 ### Windows
 
-Recommended installation methods:
+Hornet on Windows can be installed using:
 
-- [Docker image](using_docker.md)
+- [The docker image](https://wiki.iota.org/hornet/getting_started/using_docker).
 
-Other installation methods:
+It can also be installed using:
 
-- [Prebuilt binary files](hornet_apt_repository.md#pre-built-binaries)
+- [Prebuilt binary files](hornet_apt_repository.md#pre-built-binaries).
 
 ## Configuration
 
 Hornet uses several JSON configuration files that can be adjusted based on your deployment and use cases:
```

Comment:
```suggestion
Hornet uses two JSON configuration files that you can tweak based on your deployment requirements:
```

Reply:
I added "several" as a cover in case there were more since the original doc was published👌